app/src/main/java/com/cornellappdev/uplift/models/FacilityInfoModels.kt

```diff
@@ -41,16 +45,49 @@ data class BowlingInfo(
     val shoeRental: String
 )
 
+/**
+ * Representation type of accessibility there is at the facility
+ */
+//enum class AccessibilityType {
+//    WHEELCHAIR
+//}
+/**
+ * Representation type of Equipment each is
+ */
+//enum class EquipmentType{
+//    CARDIO,
+//    RACKS_AND_BENCHES,
+//    SELECTORIZED,
+//    MULTI_CABLE,
+//    FREE_WEIGHTS,
+//    MISCELLANEOUS,
+//    PLATE_LOADED,
+//}
+
+/** An [EquipmentField] is all of the equipment that exist at a gym facility */
+data class EquipmentField(
+    /** The title of this equipment grouping. (e.g. "Cardio Machines") */
+    val id : String,
+    val accessibility: AccessibilityType?,
+    val name: String,
+    val facilityId: Int,
+    val quantity: Int
+)
```

Comment:
Ah okay, this class seems like a much better solution to the `Pair` crap I was doing before!